src/config.hpp

```diff
@@ -55,3 +60,5 @@ struct config_struct {
 extern const config_struct config;
 
 extern void init_config();
+
+const std::string config_file = "config.toml";
```

Comment:
I think we'll want to look for the config file in multiple location, probably something like:
- A central system location (e.g. `/etc/clvk.conf`
- The user's home directory (e.g. `~/.config/clvk.conf`). 
- The current directory

Doing this in a portable way is the challenge but we can start by adding logic to look for the file in a list of locations, we'll definitely need that.

Also, I think we probably don't want to use the `.toml` extension. We're only using a subset of toml and that's an implementation detail. I'd suggest `clvk.conf`.

Reply:
done in this order
1. env var
2. /etc/clvk.conf
3. ~/.config/clvk.conf
4. current dir

Reply:
> done in this order
> 
> 1. env var
> 2. /etc/clvk.conf
> 3. ~/.config/clvk.conf
> 4. current dir

I'm not sure about this order.
I would have the `current dir` in second position and `/etc/clvk.conf` as the 4th.

Reply:
done!